.github/workflows/trivy.yml

```diff
@@ -36,7 +36,7 @@ jobs:
     needs: export-registry
     env:
       REGISTRY: ${{ needs.export-registry.outputs.registry }}
-    runs-on: ubuntu-20.04
+    runs-on: ubuntu-latest

```

Comment:
Hi Britania! Would you mind adding a comment saying that the `latest` tag points to the latest LTS release of Ubuntu per docker hub?